includes/fbproduct.php

```diff
@@ -617,6 +622,11 @@ public function prepare_variants_for_item( &$product_data ) {
 			// For each product field type, pull the single variant
 			foreach ( $variant_names as $original_variant_name ) {
 
+				// don't handle any attributes that are designated as Commerce attributes
+				if ( in_array( str_replace( array( 'attribute_' ), '', strtolower( $original_variant_name ) ), Products::get_distinct_product_attributes( $this->woo_product ), true ) ) {
```

Comment:
Did you intend to have multiple needles on the `str_replace`?

Reply:
Nah that's just left over from an earlier approach - removed in 0c4713234b32c857ec05d77f767b05ad285eae67